main.py

```diff
@@ -22,88 +22,91 @@ def __init__(self, username, password,token):
+        self.header['Referer'] = url
+        session_id = login_get.cookies.get("sessionid")
+        csrf_token = login_get.cookies.get("csrftoken")
         cookie_full = f"csrftoken={csrf_token}; sessionid={session_id}"
+
         self.header["Cookie"] = cookie_full
         req = requests.post(url=url, data=data, headers=self.header)
         print("The login post cookies are: ")
         print(req.cookies)
         print(req.content)
         # print("The login get cookies are: ")
-        # print(login_get.cookies)
+        # print(login_get.cookies
+
+    def get_req(self, object, offset=None, limit=None):
 
-    def get_req(self, object):
-        #get the object for ULR
         g1 = requests.get('https://wger.de/api/v2/')
-        g = requests.get(url=g1.json()[object], headers=self.header)
-        if g.status_code == 200:
-            return g.json()
+        url = g1.json().get(object)
+        if url != None:
+            req1 = requests.get(url=url, headers=self.header)
+            count = req1.json().get('count')
+            url1 = url + f"?limit={count}&offset=0"
+            g = requests.get(url=url1, headers=self.header)
+            if g.status_code == 200:
+                return g.json()
+            else:
+                return g.status_code, "Couldn't make the request!"
         else:
-            return "Couldn't make the request"
+            return False, 'Invalid url'
 
     def post_req(self,object, data):
 
         g1 = requests.get('https://wger.de/api/v2/')
         url1 = g1.json().get(object)
+
         if url1 != None:
             req = requests.post(url=url1,data=data,headers=self.header)
             if req.status_code == 201:
-                return f'The post request for {object} was done succusefuly'
+                return req, f'The post request for {object} was done succusefuly'
             else:
-                return f'Error {req.status_code}'
+                return {req.status_code}, 'Error  {req.status_code}'
         else:
-            return 'Invalid url'
-
-
+            return False, 'Invalid url'
 
     def delete_req(self,object, id):
+
         g1 = requests.get('https://wger.de/api/v2/')
         url1 = g1.json().get(object)
-        url1 = f"{url1}/{id}"
         if url1 != None:
-            req = requests.delete(url=url1, headers=self.header)
-            return req
-
-
-
-
-    def get_ingredients(self, limit=20, offset=0):
-        url = f'https://wger.de/api/v2/ingredient/?limit={limit}&offset={offset}'
-        r = requests.get(url=url)
-        if r.status_code == 200:
-            return r.json()['results']
+            url = f"{url1}/{id}"
+            req = requests.delete(url=url, headers=self.header)
+            if req.status_code not in (200,202,204):
+                return req.status_code, f"The delete for id {id} couldn't be performed"
+            return True, f"The delete for {object} with id {id} was performed" \
```

Comment:
Here you shouldn't return 'True' and the message, but the Request response + message(as you did in all the other methods).